dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings.dart

```diff
@@ -14,12 +14,12 @@ import 'package:intl/intl.dart' as intl;
 import 'stock_strings_en.dart';
 import 'stock_strings_es.dart';
 
-/// Callers can lookup localized strings with an instance of StockStrings returned
-/// by `StockStrings.of(context)`.
+/// Callers can lookup localized strings with an instance of StockStrings
+/// returned by `StockStrings.of(context)`.
```

Comment:
cc @christopherfujino if you have any pointers, I think you encountered this in the past in https://github.com/flutter/flutter/pull/145564, and said you "fixed analysis and checked it in" but I don't know what that means in this context

Reply:
I think I gave up on this because it turned into a yak shave. I would recommend just reverting the diffs to three dart files here as a workaround.

I believe @andrewkolos has a PR open that would fix this, although as I recall this is a subtly tricky problem to solve.

Reply:
Sounds good, I'll revert

Reply:
(FYI) I filed https://github.com/flutter/flutter/issues/149386 and will be trying to resolve this.